src/main/java/com/wojciechbarwinski/demo/legendary_warehouse/EpicBoarGameShopClient.java

```diff
@@ -0,0 +1,16 @@
+package com.wojciechbarwinski.demo.legendary_warehouse;
+
+
+import com.wojciechbarwinski.demo.legendary_warehouse.dtos.ShipmentInfo;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@FeignClient(name = "shipmentClient", url = "${ebgs.shipment.url")
+public interface EpicBoarGameShopClient {
+
+    @PostMapping("/shipment")
```

Comment:
To Ci działa? 
W propertasach masz:
ebgs.shipment.url=http://localhost:8080/shipment

Więc tutaj powinno się to ewaluować do http://localhost:8080/shipment/shipment